src/ButtonCatalog.js

```diff
@@ -16,11 +17,16 @@ const ButtonCatalog = () => {
       docsLink='https://material.io/components/web/catalog/buttons/'
       sourceLink='https://github.com/material-components/material-components-web/tree/master/packages/mdc-button'
       demos={<ButtonDemos />}
+      initialConfig={ButtonConfig}
+      {...props}
     />
   );
-}
+};
 
-export class ButtonHero extends Component {
+
+// This is retained for the material experiment and will be dead code that we can remove when
```

Comment:
whats the status of this?

Reply:
Still required. I'll come back and clean this Legacy classes when the experiment is complete.